include/experimental/__p0009_bits/layout_left.hpp

```diff
@@ -108,6 +109,31 @@ class layout_left::mapping {
         */
     }
 
+#if MDSPAN_HAS_CXX_17
+    MDSPAN_TEMPLATE_REQUIRES(
+      class _Mapping,
+      /* requires */ (
+        detail::__is_layout_left_padded_mapping<_Mapping>::value
```

Comment:
Ah, that makes sense now.  Would you consider adding this comment to the constructor, explaining why it is constrained that way?  This could be helpful for implementers.  Thanks!

Reply:
I also added a note for `operator==`